### PR TITLE
Refactor QuizModelTest and QuizzesServiceTest

### PR DESCRIPTION
**Pull request by miron97**
#263 

- [x] QuizModelTest | total time ~3 hours

- [x] QuizzesServiceTest | total time ~ 4,5 hours

### CODE REVIEW

**Comment by miron97** [comment]
@malinink PR ready for review

File: src/test/java/easytests/core/services/QuizzesServiceTest.java [hunk 1]
```diff
@@ -3,26 +3,29 @@
...
 import easytests.core.options.QuizzesOptionsInterface;
 import easytests.core.services.exceptions.DeleteUnidentifiedModelException;
+import easytests.support.QuizzesSupport;
 import org.junit.Assert;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
 import org.junit.runner.RunWith;
 import static org.mockito.BDDMockito.*;
+
```
Comment (malinink) [inline src/test/java/easytests/core/services/QuizzesServiceTest.java]: remove empty lines from import

File: src/test/java/easytests/core/services/QuizzesServiceTest.java [hunk 2]
```diff
@@ -37,300 +40,273 @@
...
 
     @Test
-    public void testFindPresentModel() throws Exception {
-
-        final Integer id = 1;
-        final QuizEntity quizEntity = this.createQuizEntityMock(id, "test", 1);
-        given(this.quizzesMapper.find(id)).willReturn(quizEntity);
-
-        final QuizModelInterface quizModel = this.quizzesService.find(id);
+    public void testFindAllWithOptions() throws Exception {
```
Comment (malinink) [inline src/test/java/easytests/core/services/QuizzesServiceTest.java]: Put that method after `testFindAllAbsentList`
We should try to code tests same

File: src/test/java/easytests/core/services/QuizzesServiceTest.java [hunk 3]
```diff
@@ -37,300 +40,273 @@
...
+        verifyNoMoreInteractions(this.quizzesMapper);
     }
 
     @Test
-    public void testDeleteIdentifiedModel() throws Exception {
-
-        final QuizModelInterface quizModel = this.createQuizModel(1, "test", 1);
-
-        this.quizzesService.delete(quizModel);
+    public void testSaveEntitiesList() throws Exception {
```
Comment (malinink) [inline src/test/java/easytests/core/services/QuizzesServiceTest.java]: Models ?

File: src/test/java/easytests/core/services/QuizzesServiceTest.java [hunk 4]
```diff
@@ -37,300 +40,273 @@
...
+                quizEntityCaptor.getAllValues());
     }
 
     @Test
-    public void testDeleteModelsList() throws Exception {
-
-        final QuizModelInterface quizModelFirst = this.createQuizModel(2, "test2", 1);
-        final QuizModelInterface quizModelSecond = this.createQuizModel(3, "test3", 2);
-
+    public void testSaveEntitiesListWithOptions() throws Exception {
```
Comment (malinink) [inline src/test/java/easytests/core/services/QuizzesServiceTest.java]: !Entities

**Comment by miron97** [comment]
@malinink ready for review

**Comment by malinink** [comment]
@miron97 
Make additional PR for fixing 
```
        return super.getConfigurationBuilder()
                .ignoreProperty("issue")
                .ignoreProperty("testee");
```
Ignore Property in model test.

**Pull request merged**